modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaLanguageUtilClassCheck.testjava

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import java.util.ArrayList;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaLanguageUtilClassCheck {
+
+	public void firstMethod() {
+
+		// First Case test
```

Comment:
@TamyrisTorres , which is the test case? Can you be more specific? Like: `Variable declaration test case`.